### PR TITLE
CORE-159: added tables to store app publication requests

### PR DESCRIPTION
**Pull request by slr71**

### CODE REVIEW

File: src/main/conversions/v2.26.0/c2_26_0_2019040901.clj [hunk 1]
```diff
@@ -0,0 +1,18 @@
+(ns facepalm.c2-26-0-2019040901
+  (:use [kameleon.sql-reader :only [load-sql-file]]))
+
+(def ^:private version
+  "The destination database version"
+  "2.26.0:2019049.01")
```
Comment (psarando) [inline src/main/conversions/v2.26.0/c2_26_0_2019040901.clj]: I think this version was meant to be `2.26.0:20190409.01` (missing the `0` before the `9`: `09.01`)

**Review by psarando — CHANGES_REQUESTED** [review]
Usually it seems we define 1 table per `sql` file (and likewise 1 constraints `sql` file per table), but this looks pretty tidy keeping the related tables together in one `sql` file ✨ 

Looks like there's a typo in the conversion's version number.
Also don't forget to add the new version to the `versions` data file.

File: src/main/conversions/v2.26.0/c2_26_0_2019040901.clj (hunk 1 above)
Comment (slr71) [reply src/main/conversions/v2.26.0/c2_26_0_2019040901.clj]: Ah, good catch! I'll fix it.

Comment (slr71) [reply src/main/conversions/v2.26.0/c2_26_0_2019040901.clj]: This should be fixed now.

**Comment by psarando** [comment]
I think we just need the new version added to the `data/99_version.sql` file, then this PR is good to go 👍

**Comment by slr71** [comment]
That should be fixed now. Thanks again for the reviews.

**Review by psarando — APPROVED** [review]

**Pull request merged**